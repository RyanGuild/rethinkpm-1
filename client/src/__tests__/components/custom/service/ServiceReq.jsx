import React from 'react';
import render from '../../../../__utils__/renderHelper';
import ServiceReq from '../../../../components/custom/service/ServiceReq';

test('render and snapshot', () => {
  let {container} = render(() => <ServiceReq />);
  expect(container).toMatchSnapshot();
});
