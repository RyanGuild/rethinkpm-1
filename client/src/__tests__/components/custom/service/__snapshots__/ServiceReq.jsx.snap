// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and snapshot 1`] = `
<div>
  <div>
    <p>
      Hello 
    </p>
    <form
      class="form"
    >
      <div
        class="form-group"
      >
        <label
          for="serviceType"
        >
          Type of Service
        </label>
        <select
          class="form-control"
          id="serviceType"
          name="serviceType"
        >
          <option
            disabled=""
            value=""
          >
            --Select a Service Type--
          </option>
          <option
            value="General_Maintenance"
          >
            General Maintenance
          </option>
          <option>
            HVAC
          </option>
          <option>
            Plumbing
          </option>
          <option>
            Pest Control
          </option>
          <option>
            Other
          </option>
        </select>
      </div>
      <div
        class="form-group"
      >
        <label
          for="serviceType"
        >
          Service Request Summary
        </label>
        <textarea
          class="form-control"
          col="80"
          id="serviceDiscription"
          name="serviceDiscription"
          row="6"
        />
      </div>
      <div
        class=""
      >
        <h4>
          Scheduling
        </h4>
        <p>
          Please select the times you will be available.
        </p>
      </div>
      <table
        class="table table-striped table-condensed"
      >
        <thead>
          <tr>
            <th />
            <th
              scope="col"
            >
              Morning 
              <br />
              <span
                class="small"
              >
                8-11am
              </span>
            </th>
            <th
              scope="col"
            >
              Afternoon
              <br />
              <span
                class="small"
              >
                11am-3pm
              </span>
            </th>
            <th
              scope="col"
            >
              Evening
              <br />
              <span
                class="small"
              >
                3pm-6pm
              </span>
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <th
              id="day1"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="11"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="12"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="13"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day2"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="21"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="22"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="23"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day3"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="31"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="32"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="33"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day4"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="41"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="42"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="43"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day5"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="51"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="52"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="53"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day6"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="61"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="62"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="63"
              />
            </td>
          </tr>
          <tr>
            <th
              id="day7"
              scope="row"
            >
              Day: and Date
            </th>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="71"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="72"
              />
            </td>
            <td>
               
              <input
                name="avail"
                type="checkbox"
                value="73"
              />
            </td>
          </tr>
        </tbody>
      </table>
    </form>
  </div>
</div>
`;
